plugins/keep-markup/tests/test.js

```diff
@@ -0,0 +1,88 @@
+const expect = require('chai').expect;
+const jsdom = require('jsdom')
+const { JSDOM } = jsdom
+
+require('../../../prism')
+// fake DOM
+global.self = {}
+global.self.Prism = Prism
+global.document = {}
+document.createRange = function () {
```

Comment:
`createRange` is not implemented in JSDOM: https://github.com/jsdom/jsdom/issues/317